README.rst

```diff
@@ -156,7 +156,7 @@ the form of::
 
 (pronounced "at localpart on my dot domain dot name").
 
-As when logging in, you will need to specify a "Custom server".  Specify your
+When logging in, you will need to specify a "Custom server".  Specify your
```

Comment:
This changes the meaning. The point of "as when" is to emphasise that the upcoming details apply to another scenario too.

```suggestion
As when logging in, you will need to specify a "Custom server". Specify your
```